### PR TITLE
rest: add a like test

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

File: gnocchi/tests/functional/gabbits/search.yaml [hunk 1]
```diff
@@ -64,6 +64,14 @@ tests:
       response_json_paths:
         $.`len`: 0
 
+    - name: search like id
+      POST: /v1/search/resource/generic
+      data:
+        eq:
+          id: 'faef212f%'
```
Comment (jd) [inline gnocchi/tests/functional/gabbits/search.yaml]: not sure why it works? the operator is `eq`?

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 45fd16c42d567a46517873df98441b2a64a713ba

File: gnocchi/tests/functional/gabbits/search.yaml (hunk 1 above)
Comment (sileht) [inline gnocchi/tests/functional/gabbits/search.yaml]: The operator should be like.

**Comment by sileht** [comment]
Pull-request updated, HEAD is now b0e3d5b113c2606cb9a9ddbb9221384278df692b

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 4c39e4bec0b85a39c3fa709102bedc716e2cf287

**Comment by sileht** [comment]
I haven't dig, but yeah the test fail with created_by_project_id...

**Pull request merged**